code/modules/research/xenobiology/crossbreeding/loyal.dm

```diff
@@ -0,0 +1,745 @@
+	throwforce = 0
+	w_class = WEIGHT_CLASS_TINY
+	throw_range = 1
+	throw_speed = 1
+	var/obj/item/original
+
+/obj/item/disguised_item/attack_self(mob/user)
+	to_chat(user,"<span class='notice'>You begin removing the disguise of [src]...</span>")
+	if(do_mob(user,src,20))
+		src.forceMove(original.loc) //Just to get it out of our hands
+		user.putItemFromInventoryInHandIfPossible(original, user.active_hand_index, TRUE)
+		to_chat(user, "<span class='notice'>You reveal [original]!</span>")
+		qdel(src)
+
+/obj/item/slimecross/loyal/pink
+	colour = "pink"
+	effect_desc = "Makes an item very pleasing to have, and improves the holder's mood."
+	loyaleffect = /datum/component/loyal_effect/pink
+
+/datum/component/loyal_effect/pink
+	prefix = "soothing"
+
+/datum/component/loyal_effect/pink/Initialize(obj/item/attached_item)
+	. = ..()
+	START_PROCESSING(SSobj,src)
+
+/datum/component/loyal_effect/pink/Destroy()
+	STOP_PROCESSING(SSobj,src)
+	return ..()
+
+/datum/component/loyal_effect/pink/process()
+	var/humanfound = null //Copied from stabilized extract. Which is uh, questionable.
+	if(ishuman(attached.loc))
+		humanfound = attached.loc
+	if(ishuman(attached.loc.loc)) //Check if in backpack.
+		humanfound = (attached.loc.loc)
+	if(!humanfound)
+		return
+	var/mob/living/carbon/human/H = humanfound
+	SEND_SIGNAL(H, COMSIG_ADD_MOOD_EVENT, "soothing", /datum/mood_event/soothing, attached)
+
+/obj/item/slimecross/loyal/gold
+	colour = "gold"
+	effect_desc = "Makes an item responsive to hand motions, and able to be commanded to approach."
+	loyaleffect = /datum/component/loyal_effect/gold
+
+/datum/component/loyal_effect/gold
+	prefix = "responsive"
+
+/obj/item/slimecross/loyal/oil
+	colour = "oil"
+	effect_desc = "Makes an item slippery."
+	loyaleffect = /datum/component/loyal_effect/oil
+
+/datum/component/loyal_effect/oil
+	prefix = "slippery"
+
+/datum/component/loyal_effect/oil/Initialize(obj/item/attached_item)
+	. = ..()
+	if(attached_item)
+		attached_item.AddComponent(/datum/component/slippery, 80) //It's just soap
+
+/obj/item/slimecross/loyal/black
+	colour = "black"
+	effect_desc = "Transforms the item into an obediant black slime. Upon death, the slime will revert back into the object."
+	loyaleffect = /datum/component/loyal_effect/black
+
+/obj/item/slimecross/loyal/black/afterattack(obj/item/target,mob/user,proximity)
+	if(!proximity || target.GetComponent(/datum/component/loyal_effect))
+		return
+	if(!..())
+		return
+	var/mob/living/simple_animal/slime/blackslime = new /mob/living/simple_animal/slime(target.loc,FALSE, "black", TRUE)
+	blackslime.Friends[user] = 10 //This should be obediant enough
+	target.forceMove(blackslime)
+	//target.RegisterSignal(blackslime, COMSIG_MOB_DEATH, .proc/drop_attached)
+
+//mob/living/simple_animal/slime/proc/drop_attached(datum/source, gibbed)
+
+/datum/component/loyal_effect/black
+	prefix = "transformative"
+
+/obj/item/slimecross/loyal/lightpink
```

Comment:
I need a way to test this one, I couldn't connect with two clients to my test server.